lib/error-matcher.js

```diff
@@ -23,12 +23,21 @@ module.exports = (function () {
   util.inherits(ErrorMatcher, EventEmitter);
 
   ErrorMatcher.prototype._goto = function () {
-    var file = this.cwd + path.sep + this.currentMatch[0].file;
-    if (!this.currentMatch[0].file) {
+    var file = this.currentMatch[0].file;
+    if (file) {
+      var oldDir = process.cwd();
+      process.chdir(this.cwd);
```

Comment:
The global behavior isn't changed by my patch since I change the cwd back after the realpathSync call. But `fs.isAbsolute` actually looks like a good and clean solution, so to be safe let's get rid of `process.chdir` :). It also makes it clearer what's going on.